worktree.go

```diff
@@ -167,6 +170,61 @@ func (w *Worktree) getMode(fi billy.FileInfo) os.FileMode {
 	return object.FileMode
 }
 
+const gitmodulesFile = ".gitmodules"
+
+func (w *Worktree) Submodules() (Submodules, error) {
+	l := make(Submodules, 0)
```

Comment:
make this after calling readGitmodulesFile, and make it with the size of the m.Submodules,